Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/KendoRemoteDataController.cs

```diff
@@ -54,6 +58,29 @@ public JsonResult GetUsersContaining(string userFilter)
             return this.Json(result, JsonRequestBehavior.AllowGet);
         }
 
+        [AjaxOnly]
+        public JsonResult GetProblemsContaining(string text)
+        {
+            var problems = this.problemsData.GetAll().Take(DefaultItemsToTake);
+
+            if (!string.IsNullOrWhiteSpace(text))
+            {
+                problems = this.problemsData
+                    .GetAll()
+                    .Where(p => p.Name.Contains(text))
+                    .Take(DefaultItemsToTake);
+            }
+
+            var result = problems
+                .Select(p => new DropdownViewModel
```

Comment:
I personally like this formatting, but I think `Select` should be on the above like. Similarly to `ZippedTestsParser`, line 84.